src/Symfony/Component/Validator/Constraints/IbanValidator.php

```diff
@@ -57,7 +57,7 @@ class IbanValidator extends ConstraintValidator
         'CH' => 'CH\d{2}\d{5}[\dA-Z]{12}', // Switzerland
         'CI' => 'CI\d{2}[A-Z]{1}\d{23}', // Ivory Coast
         'CM' => 'CM\d{2}\d{23}', // Cameron
-        'CR' => 'CR\d{2}\d{3}\d{14}', // Costa Rica
+        'CR' => 'CR\d{2}0\d{3}\d{14}', // Costa Rica
```

Comment:
not sure this is security related, thus needs to be changed on lower branches. I'd prefer the BC variant.

Reply:
Accepting an invalid IBAN is a bug. There's no  BC to preserve here.

Reply:
Right.. this wont lead to crashing behavior. My bad :+1:

Reply:
https://bank.codes/iban/structure/costa-rica/  implies it can be any kind of number btw... not sure in practice it's always `0`.

Reply:
@ro0NL https://www.iban.com/calculate-iban.html - this site does not give you ability to choose Reserve Number. It automatically set it 0

Reply:
:+1:

Reply:
@ro0NL Ok I leave it like that and point it to 2.7